src/components/DiceRoller.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div>
+        <button @click="rollDice">Roll Dice</button>
+        <h3>{{numDice}}d{{dieType}}+{{modifier}} <span v-html="this.results"></span></h3>

```

Comment:
This might result in `1d8+-5` with a negative modifier. Would be nicer to make the `+` conditional on it being a positive number. Would also be nice if it was omitted entirely for `0` as `1d8` is enough.